src/vm/i386/asmhelpers.S

```diff
@@ -815,6 +815,13 @@ PATCH_LABEL ExternalMethodFixupPatchLabel
 NESTED_END ExternalMethodFixupStub, _TEXT
 
 #ifdef FEATURE_READYTORUN
+NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
+    CHECK_STACK_ALIGNMENT
+    call  eax
+    add   esp, 12
+    ret
+NESTED_END DynamicHelperArgsStub, _TEXT
```

Comment:
@parjong I agree that to make it unwindable, it requires a special treatment. But it is not necessary to create an EBP frame there and the way you suggested it, it would not work anyways (there is no CFI annotation, so the unwinder would have no idea where to get the EBP). When there are no annotations, the default is the CFA to be ESP + 4, which means the unwinder gets the EIP of the caller frame from [CFA-4], which is [ESP] and then assigns CFA to ESP.
So I would suggest modifying the DynamicHelperArgsStub as follows. The `.cfi_def_cfa_offset 16` tells the unwinder that the CFA at that point in the code is at ESP+16. 
```asm
NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
    .cfi_def_cfa_offset 16
    CHECK_STACK_ALIGNMENT
    call  eax
    add   esp, 12
    ret
NESTED_END DynamicHelperArgsStub, _TEXT

```